htmd/queues/lsfqueue.py

```diff
@@ -21,31 +21,40 @@ class LsfQueue(SimQueue, ProtocolInterface):
 
     Parameters
     ----------
+    version : [9, 10], int, default=9
+        LSF major version
     jobname : str, default=None
         Job name (identifier)
-    queue : str or list of str, default=None
-        The queue or list of queues to run on. If list, it attempts to submit the job to the first queue listed. If that
-        queue cannot be used, the next queue listed is considered and this process is repeated.
+    queue : list, default=None
+        The queue or list of queues to run on. If list, it attempts to submit the job to the first queue listed
     app : str, default=None
         The application profile
     ngpu : int, default=1
         Number of GPUs to use for a single job
+    gpu_options : dict, default=None
+        Number of GPUs to use for a single job(dict format: {['mode': ['shared', 'exclusive_process'],]['mps': ['yes',
+        'no'],]['j_exclusive': ['yes', 'no']]})
     ncpu : int, default=1
         Number of CPUs to use for a single job
     memory : int, default=4000
         Amount of memory per job (MiB)
     walltime : int, default=None
         Job timeout (hour:min or min)
-    envvars : str, default='ACEMD_HOME'
-        Envvars to propagate from submission node to the running node (comma-separated)
-    prerun : list of strings, default=None
-        Shell commands to execute on the running node before the job (e.g. loading modules)
-    resources : list of strings, default=None
+    resources : list, default=None
         Resources of the queue
-    outputstream : str, default='slurm.%N.%j.out'
+    outputstream : str, default='lsf.%J.out'
         Output stream.
-    errorstream : str, default='slurm.%N.%j.err'
+    errorstream : str, default='lsf.%J.err'
         Error stream.
+    datadir : str, default=None
+        The path in which to store completed trajectories.
+    trajext : str, default='xtc'
+        Extension of trajectory files. This is needed to copy them to datadir.
+    envvars : str, default='ACEMD_HOME'
+        Envvars to propagate from submission node to the running node (comma-separated)
+    prerun : list, default=None
+        Shell commands to execute on the running node before the job (e.g. loading modules)
+
```

Comment:
FYI `ProtocolInterface` keeps track of the order with which you declare the parameters. So if you want to switch the order in the documentation just declare them in that order in the `__init__` ;)

Reply:
this reordering was just because I created the docstring manually last time instead of generating it with the _printDocString you showed me today

Reply:
yes I just mentioned in in case you prefered the old order

Reply:
I'm actually OK with the order being generated from the `_printDocString` :)